propagation.go

```diff
@@ -128,21 +131,27 @@ func extractTraceContext(opaqueCarrier interface{}) (SpanContext, error) {
 		pickupW3CTraceContext(http.Header(c), &spanContext)
 	}
 
+	// Iterate over the headers, look for Instana headers and try to parse them.
+	// In case of error interrupt, iteration and return it.
 	err := carrier.ForeachKey(func(k, v string) error {
 		var err error
 
 		switch strings.ToLower(k) {
 		case FieldT:
 			spanContext.TraceIDHi, spanContext.TraceID, err = ParseLongID(v)
 			if err != nil {
+				sensor.logger.Debug("extract trace context ", FieldT, "=", v, " : ", err)
```

Comment:
Do we still need these log lines?

Reply:
Without it, there is no way to see the original error from `ParseLongID `. Assuming this is debugging level, I would keep it. Anyway, this error should never exist in a healthy environment.

Reply:
I thought once more, let's remove it :)

Reply:
You’re right, this might be helpful while debugging, however in case of handling concurrent requests this log line may be far away from the one where we report an error and if, for example, the header level is a non-hex string, there is no way to correlate these two messages. This is why I suggested to put the malformed value into the error log at first place. But I guess logging the parsed values is also fine, at least we know which piece of context is missing